src/sentry/api/endpoints/organization_sampling_project_rates.py

```diff
@@ -0,0 +1,111 @@
+from collections.abc import Mapping, MutableMapping
+from typing import Any
+
+from django.db import router, transaction
+from rest_framework import serializers
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases import OrganizationEndpoint, OrganizationPermission
+from sentry.api.paginator import OffsetPaginator
+from sentry.api.serializers import Serializer, serialize
+from sentry.constants import TARGET_SAMPLE_RATE_DEFAULT, ObjectStatus
+from sentry.models.options.project_option import ProjectOption
+from sentry.models.project import Project
+
+OPTION_KEY = "sentry:target_sample_rate"
+
+
+class GetSerializer(Serializer):
+    """Serializer for OrganizationSamplingProjectRatesEndpoint.get"""
+
+    def get_attrs(self, item_list, user, **kwargs) -> MutableMapping[Any, Any]:
+        options = ProjectOption.objects.get_value_bulk(item_list, OPTION_KEY)
+        # NOTE: `get_value_bulk` does not resolve defaults. The default does not
+        # depend on epochs, so we can speed this up by using the constant.
+        return {
+            item: value if value is not None else TARGET_SAMPLE_RATE_DEFAULT
+            for item, value in options.items()
+        }
+
+    def serialize(self, obj: Any, attrs: Any, user, **kwargs) -> Mapping[str, Any]:
+        return {"id": obj.id, "sampleRate": attrs}
+
+
+class PutSerializer(serializers.Serializer):
+    """Serializer for OrganizationSamplingProjectRatesEndpoint.put"""
+
+    id = serializers.IntegerField(required=True)
+    sampleRate = serializers.FloatField(required=True, min_value=0, max_value=1)
+
+
+@region_silo_endpoint
+class OrganizationSamplingProjectRatesEndpoint(OrganizationEndpoint):
+    """Bulk endpoint for managing project sampling rates."""
+
+    owner = ApiOwner.TELEMETRY_EXPERIENCE
+
+    publish_status = {
+        "GET": ApiPublishStatus.PRIVATE,
+        "PUT": ApiPublishStatus.PRIVATE,
+    }
+
+    permission_classes = (OrganizationPermission,)
+
+    def get(self, request: Request, organization) -> Response:
+        """
+        List Sampling Rates for Projects
+        ````````````````````````````````
+
+        Return a list of sampling rates for projects in the organization by
+        project ID.
+
+        :pparam string organization_id_or_slug: the id or slug of the
+            organization.
+        :auth: required
+        """
+
+        # NOTE: This fetches all projects in the organization. We do not filter
+        # to projects the org member has access to as the sample rate and
+        # project ID do not constitute sensitive information.
+        queryset = Project.objects.filter(
+            organization=organization,
+            status=ObjectStatus.ACTIVE,
+        )
+
+        return self.paginate(
```

Comment:
Are we omitting the default_per_page here on purpose?

Reply:
We're using the default of 100 (via [`PAGINATION_DEFAULT_PER_PAGE`](https://github.com/getsentry/sentry/blob/e9a8f767d8c12ddd3b7b2446a6bfe77f96c4decb/src/sentry/api/base.py#L80)). Is there a different default you'd suggest?

Reply:
Nope, just checking that that is a consciuos decision - thx.